debian/postinst

```diff
@@ -67,8 +68,31 @@ upgrade_to_status_cache() {
     ua status 2>&1 > /dev/null
 }
 
+is_release_upgrade() {
+    # Return 0 if the package version we upgrade from does not match the
+    # current series (e.g. 14.04)
+    if test "${2#*$VERSION_ID}" = ${2}; then
```

Comment:
Calling `test` directly is slightly odd; using `if [ ... ]` is more standard.

Reply:
dropped test and used your case statement suggestions